From: Michael Tokarev <mjt@tls.msk.ru>
Subject: [PATCH trivial] do not fail build if SIGPWR is not defined
To: busybox@busybox.net

Apparently, some *BSD variants (and maybe some other OSes) does not define
SIGPWR signal.  So since commit 760fc6debcba8c, busybox fails to build on
such platforms.  Fix this.

Signed-off-By: Michael Tokarev <mjt@tls.msk.ru>

diff --git a/init/init.c b/init/init.c
index d29328c..992a21e 100644
--- a/init/init.c
+++ b/init/init.c
@@ -1135,7 +1137,9 @@ int init_main(int argc UNUSED_PARAM, char **argv)
 		struct sigaction sa;
 
 		bb_signals(0
+#ifdef SIGPWR
 			+ (1 << SIGPWR)  /* halt */
+#endif
 			+ (1 << SIGUSR1) /* halt */
 			+ (1 << SIGTERM) /* reboot */
 			+ (1 << SIGUSR2) /* poweroff */
